dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -125,7 +125,38 @@ This is the creative that your Ad Ops team needs to upload to the ad server (it'
 {: .alert.alert-success :}
 You can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.html).
 
-{% include dev-docs/amp-creative.md %}
+{: .alert.alert-danger :}
+Replace **MACRO** in the code below with the appropriate macro for the ad server you're using. For example, for DFP replace **MACRO** with **PATTERN**.
+
+{% highlight html %}
+
+    <script src="https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
+    <script>
+      var ucTagData = {};
+      ucTagData.adServerDomain = "";
+      ucTagData.pubUrl = "%%MACRO:url%%";
+
+      // if DFP, use this param
+      ucTagData.targetingMap = %%MACRO:TARGETINGMAP%%;
+
+      // if not DFP, use these params
```

Comment:
Why are these params in this version of the UC tag but not the previous version?

Reply:
@bretg Can you clarify which params you are referring to?  If it's the `targetingMap`, there was some rewrite to the logic in the creative.js file to support handling this param with respect to the original set.

Reply:
Can we use a single, consistent universal creative snippet across all examples?